content/blog/2021/08/DeployingCamelRouteInAWSLambda/index.md

```diff
@@ -0,0 +1,210 @@
+---
+title: "Deploying a Camel Route in AWS Lambda using Quarkus"
+date: 2021-08-19
+authors: [ravishankarhassain]
+categories: ["AWS", "Lambda","Camel Quarkus"]
```

Comment:
I'm wondering if it's useful to have categories like "AWS" and "Lambda". Maybe "AWS Lambda" ? Or let's see what other think.